cloudnet-common/src/main/java/de/dytanic/cloudnet/common/JavaVersion.java

```diff
@@ -0,0 +1,63 @@
+package de.dytanic.cloudnet.common;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum JavaVersion {
+    JAVA_7(7, 51D, "Java 7"),
```

Comment:
Java 7? CloudNet can't run in this either.

Reply:
:man_shrugging: